### DISCUSSION TITLE
Page continuously refreshes

### DISCUSSION THREAD

**Discussion by pbrianmackey** [compodoc/compodoc#1161] [General]
I just did a fresh installation of compodoc on an existing project.  The project is quite large.  The issue is compodoc keeps refreshing itself as if I had hit F5 refresh on my browser about every 5 seconds.  I followed the documentation for setup and I run `compodoc -p tsconfig.dos.json -s` to serve.

My only guess is that compodoc can't handle the massive size of the project at hand.  Any ideas?

**Comment by vogloblinsky** [comment]
Hi, is there an error in JavaScript console ?

**Comment by dberthia** [comment]
Did you ever identify a workaround for this? I just enabled compodoc on my project, and am using the -s -o flags. When the docs launch in the browser (Edge in my case), there's a "Live reload enabled" message in the console, and the page is reloading every few seconds. I'm running localhost on Angular 13.3 and node 18.15. Is there some option to disable live reload? I'm not seeing one. I think something like "ng serve --live-reload false" would fix it, so maybe add that option as a flag?

**Reply by vogloblinsky** [reply]
Strange, -s just enable a static http-server and -o to directly open it.
